### PR TITLE
Update build instructions, Windows JDK16 compiles with VS2019

### PR DESCRIPTION
**Pull request by pshipton**
Also remove some freemarker remnants from jdk8.

### CODE REVIEW

**Comment by ChengJin01** [comment]
These two line for vs2017 might need to be updated accordingly:
```
regsvr32 "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\DIA SDK\bin\msdia140.dll"
regsvr32 "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\DIA SDK\bin\amd64\msdia140.dll"
```

**Comment by pshipton** [comment]
Hopefully I've got all the references now.

File: doc/build-instructions/Build_Instructions_V16.md [hunk 1]
```diff
@@ -391,19 +391,19 @@ cd /cygdrive/c/temp
...
 ```
 
-- Before installing Visual Studio, change the permissions on the installation file by running chmod u+x vs2017.exe.
-- Install Visual Studio by running the file `vs2017.exe` (There is no special step required for downloading/installing VS2017. Please follow the guide of the downloaded installer to install all required components, especially for VC compiler).
+- Before installing Visual Studio, change the permissions on the installation file by running `chmod u+x vs2019.exe`.
+- Install Visual Studio by running the file `vs2019.exe` (There is no special step required for installing. Please follow the guide of the installer to install all desired components, the C++ compiler is required).
 
 Not all of the shared libraries that are included with Visual Studio are registered during installation.
-In particular, the `msdia140.dll`(VS2017) library must be registered manually by running command prompt as administrator.  To do so, execute the following from a command prompt:
+In particular, the `msdia140.dll` library must be registered manually by running command prompt as administrator.  To do so, execute the following from a command prompt:
```
Comment (keithc-ca) [inline doc/build-instructions/Build_Instructions_V16.md:402]: nit: library -> libraries (there are two of them)

File: doc/build-instructions/Build_Instructions_V8.md [hunk 2]
```diff
@@ -405,13 +405,11 @@ cd /cygdrive/c/temp
 
 - Run the following commands:
 ```
-wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz
 wget http://download.savannah.gnu.org/releases/freetype/freetype-2.5.3.tar.gz
 ```
 
-- To unpack the Freemarker and Freetype compressed files, run:
+- To unpack the Freetype compressed files, run:
```
Comment (keithc-ca) [inline doc/build-instructions/Build_Instructions_V8.md:411]: compressed files -> archive

**Comment by pshipton** [comment]
Fixed the review comments, and a few additional tweaks.

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
@ChengJin01 Any further suggestions?

**Comment by ChengJin01** [comment]
I am OK with the updated changes

**Pull request merged**